src/widgetastic_patternfly/__init__.py

```diff
@@ -2027,3 +2034,204 @@ def is_displayed(self):
+    COUNT = './/span[contains(@class, "card-pf-aggregate-status-count")]'
+    TITLE_ANCHOR = '/a'
+
+    BODY = './div[contains(@class, "card-pf-body")]'
+    NOTIFICATION = ('./p[contains(@class, "card-pf-aggregate-status-notifications")]'
+                    '//span[contains(@class, "card-pf-aggregate-status-notification")]')
+
+    ACTION_ANCHOR = ParametrizedLocator('.//a[@title={@action_title|quote} '
+                                        'or @data-original-title={@action_title|quote}]')
+
+    def __init__(self, parent, name, locator=None, action_title=None, logger=None):
+        """Constructor, using name, can specify locator
+
+        Args:
+            name: string name of the status card, displayed with count in top line
+            action_title: title attribute for the anchor of the action link in notification block
+                            In the patternfly ref, this is an 'add' action with icon
+        """
+        Widget.__init__(self, parent=parent, logger=logger)
+        self.name = name
+        self.locator = locator or self.ROOT
+        self.action_title = action_title
+
+    def __locator__(self):
+        return self.locator
+
+    @property
+    def _title(self):
+        """tool for local methods to get the title element"""
+        return self.browser.element(self.TITLE, parent=self)
+
+    @property
+    def _body(self):
+        """tool for local methods to get the body element"""
+        return self.browser.element(self.BODY, parent=self)
+
+    @property
+    def count(self):
+        """count in the title
+
+        Returns:
+            None if no count element is found
+            int count from the element
+        """
+        try:
+            return int(
+                self.browser.text(
+                    self.browser.element(self.COUNT, parent=self._title)
+                )
+            )
+        except NoSuchElementException:
+            return None
+
+    @property
+    def icon(self):
+        """icon of the title
+
+        Returns:
+            None if no icon is found in the title element
+            PFIcon constant if icon found
+        """
+        try:
+            return PFIcon.icon_from_element(element=self._title, browser=self.browser)
+        except NoSuchElementException:
+            return None
+
+    @property
+    def notifications(self):
+        """read method for the status notifications in the body of the card
+
+        Returns
+            list of notification elements, empty when there are none
+        """
+        try:
+            notes = self.browser.elements(self.NOTIFICATION, parent=self._body)
+        except NoSuchElementException:
+            return []
+        return [
+            StatusNotification(parent=self, note_element=note, logger=self.logger)
+            for note in notes
+        ]
+
+    def read(self):
+        items = {
+            k: getattr(self, k)
+            for k in ['icon', 'count', 'name']
```

Comment:
just writing the full dict out here might be a little more readable and comprehensible